src/docbuilder/rustwide_builder.rs

```diff
@@ -93,15 +94,26 @@ impl RustwideBuilder {
             .map(|t| Cow::Owned(t))
             .unwrap_or_else(|_| Cow::Borrowed("nightly"));
 
+        let cpu_limit = std::env::var("DOCS_RS_BUILD_CPU_LIMIT")
+            .map(|limit| limit.parse::<u32>().expect("invalid DOCS_RS_BUILD_CPU_LIMIT"));
```

Comment:
Whoops, I never actually tested the suggested change. Maybe try this?

```suggestion
        let cpu_limit = std::env::var("DOCS_RS_BUILD_CPU_LIMIT")
            .ok().map(|limit| limit.parse::<u32>().expect("invalid DOCS_RS_BUILD_CPU_LIMIT"));
```

Reply:
Just checked and this does fix the issue.